src/AppBundle/Controller/ClassesController.php

```diff
@@ -0,0 +1,127 @@
+<?php
+
+namespace AppBundle\Controller;
+
+use AppBundle\Entity\Classes;
+use AppBundle\Entity\Teachers;
+use AppBundle\Repository\ClassesRepository;
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\Route;
+use Symfony\Bridge\Doctrine\Form\Type\EntityType;
+use Symfony\Bundle\FrameworkBundle\Controller\Controller;
+use Symfony\Component\Form\Extension\Core\Type\ChoiceType;
+use Symfony\Component\Form\Extension\Core\Type\SubmitType;
+use Symfony\Component\Form\Extension\Core\Type\TelType;
+use Symfony\Component\Form\Extension\Core\Type\TextareaType;
+use Symfony\Component\Form\Extension\Core\Type\TextType;
+use Symfony\Component\HttpFoundation\Request;
+
+class ClassesController extends Controller
+{
+    /**
+     * @Route("classes", name="classes_listing")
+     */
+    public function indexAction()
+    {
+        $classes = $this->getDoctrine()
```

Comment:
Always create a relevant service and put this sort of code into service, the relevant service here should be ClassService.